docs/node_management_overview.md

```diff
@@ -2,67 +2,35 @@
 
 ## Automatic Agent Upgrade
 
-The Automatic Agent Upgrade is a policy-based node management feature that allows an org admin to create node management policies that deploy upgrade jobs to nodes and manages them autonomously. This allows the admin to ensure that all the nodes in the system are on their intended versions.
+The Automatic Agent Upgrade is a policy-based node management feature that allows an org admin to create node management policies that deploy upgrade jobs to nodes and manages them autonomously. This allows the admin to ensure that all the nodes in the system are using the intended versions.
 
-### How to set-up an Automatic Agent Upgrade policy
-
-1. Create a manifest
-    - The first step is to create a manifest using the command below and saving it to a file. In this example, the file is named `manifest.json`
-    ```
-    hzn nodemanagement manifest new > manifest.json
-    ```
-    **Note**: For more detailed information about manifests, see [here](./agentfile_manifest.md)
-
-2. Edit the manifest file
-    - Using a text editor, edit the manifest file to include all the files and versions needed for the upgrade job. The file should look similar to the example below.
-    ```
-    {
-        "softwareUpgrade": {
-            "files": [
-                "agent-install.sh",
-                "horizon-agent-linux-deb-amd64.tar.gz"
-            ],
-            "version": "latest"
-        },
-        "certificateUpgrade": {
-            "files": [
-                "agent-install.crt"
-            ],
-            "version": "latest"
-        },
-        "configurationUpgrade": {
-            "files": [
-                "agent-install.cfg"
-            ],
-            "version": "latest"
-        }
-    }
-    ```
-    **Note**: The files and versions must correspond to objects stored on the Management Hub. Use the `hzn nodemanagement agentfiles list` command to get a list of available files and versions.
-
-3. Add the manifest file to the Management Hub
+This capability utilizes existing agent artifacts that reside in Cloud Sync Service (CSS) from the edgeNodeFiles.sh installation script.
```

Comment:
Should we change capability to feature here.  In 5 above, it states 'The Automatic Agent Upgrade is a policy-based node management feature ... ' 
For consistency, it reads better if we say 'This feature utilizes... '